Source/WebKit/UIProcess/Extensions/WebExtensionControllerConfiguration.h

```diff
@@ -50,7 +51,7 @@ class WebExtensionControllerConfiguration : public API::ObjectImpl<API::Object::
     explicit WebExtensionControllerConfiguration(IsPersistent);
     explicit WebExtensionControllerConfiguration(const UUID&);
 
-    std::optional<UUID> identifier() const { return m_identifier; }
+    Markable<UUID> identifier() const { return m_identifier; }
```

Comment:
You can continue returning std::optional if it is better since Markable <-> std::optional conversion can happen implicitly. So, I would recommend using std::optional usually, and using Markable only when we store it to the member.